packages/core/useVModel/index.ts

```diff
@@ -127,16 +127,23 @@ export function useVModel<P extends object, K extends keyof P, Name extends stri
   if (passive) {
     const initialValue = getValue()
     const proxy = ref<P[K]>(initialValue!)
+    let isUpdating = false
 
     watch(
       () => props[key!],
-      v => proxy.value = cloneFn(v) as UnwrapRef<P[K]>,
+      (v) => {
+        if (!isUpdating) {
+          isUpdating = true
+          proxy.value = cloneFn(v) as UnwrapRef<P[K]>
+          setTimeout(() => isUpdating = false)
```

Comment:
Better use nextTick here

Reply:
Using nextTick instead of setTimeout is a better way, thank you for your advise, I have already test and changed the commit.